conductor/lib/nuke_utils.py

```diff
@@ -5,7 +5,20 @@
 def get_image_dirpath():
     pass
 
+def get_nuke_version():
+    '''
+    Return the full version string of the currently running nuke session. 
+    e.g. "9.0v5"
+    '''
+    return nuke.env["NukeVersionString"]
 
+def derive_docker_image(version):
```

Comment:
This function will obviously fail if any other version of nuke is used (aside from 9.0v5 and 9.0v7), so we'll need to revisit this shortly.